pkg/generators/forwarding/fluentd/templates.go

```diff
@@ -622,6 +622,19 @@ const outputLabelConfJsonParseNoretryTemplate = `{{- define "outputLabelConfJson
 	merge_json_log false
 	replace_json_log true
   </filter>
+{{ if .Target.Syslog.AddLogSource }}
+  <filter **>
+	@type record_modifier
+	<record>
+	  namespace_info      ${if record.has_key?('kubernetes'); "namespace_name=" + record['kubernetes']['namespace_name']; else nil; end}
```

Comment:
I tried this patch and checked if we can send audit logs via syslog forwarding.
But it failed to send audit logs with the following error:
```
2020-12-04 18:23:57 +0000 [warn]: dump an error event: error_class=TypeError error="no implicit conversion of nil into String" location="/usr/local/share/gems/gems/fluent-plugin-record-modifier-2.1.0/lib/fluent/plugin/filter_record_modifier.rb:174:in `+'" tag="k8s-audit.log" time=2020-12-04 18:23:57.473698000 +0000 record={"kind"=>"Event", "apiVersion"=>"audit.k8s.io/v1", "level"=>"info", "auditID"=>"45543604-2e70-44c6-ab25-6b670eac13d5", "stage"=>"ResponseComplete", "requestURI"=>"/apis/authentication.k8s.io/v1/tokenreviews", "verb"=>"create", "user"=>{"username"=>"system:serviceaccount:openshift-multus:multus", "uid"=>"a48aa3fc-eacb-4cec-934a-31a65955d6dd", "groups"=>["system:serviceaccounts", "system:serviceaccounts:openshift-multus", "system:authenticated"]}, "sourceIPs"=>["192.168.200.11"], "userAgent"=>"kube-rbac-proxy/v0.0.0 (linux/amd64) kubernetes/$Format", "objectRef"=>{"resource"=>"tokenreviews", "apiGroup"=>"authentication.k8s.io", "apiVersion"=>"v1"}, "responseStatus"=>{"code"=>201}, "requestReceivedTimestamp"=>"2020-12-04T18:23:57.473698Z", "stageTimestamp"=>"2020-12-04T18:23:57.479767Z", "annotations"=>{"authentication.k8s.io/legacy-token"=>"system:serviceaccount:openshift-multus:multus", "authorization.k8s.io/decision"=>"allow", "authorization.k8s.io/reason"=>"RBAC: allowed by ClusterRoleBinding \"multus-admission-controller-webhook\" of ClusterRole \"multus-admission-controller-webhook\" to ServiceAccount \"multus/openshift-multus\""}, "k8s_audit_level"=>"Metadata", "message"=>nil, "hostname"=>"master2.hawk2p7-1.cluster.sub.nec.test", "pipeline_metadata"=>{"collector"=>{"ipaddr4"=>"192.168.200.12", "inputname"=>"fluent-plugin-systemd", "name"=>"fluentd", "received_at"=>"2020-12-04T18:23:57.481102+00:00", "version"=>"1.7.4 1.6.0"}}, "@timestamp"=>"2020-12-04T18:23:57.473698+00:00", "viaq_index_name"=>"audit-write", "viaq_msg_id"=>"MDZmMDI3ZTItNDY3MC00ZTMwLWE3YTktNzhkNWNhZWQ2YmU0", "kubernetes"=>{}}
```
As for Audit log, the "record.has_key?('kubernetes');" doesn't work well.
Because audit record also has "kubernetes" key in its record although the value is "{}".